components/base/kinematicbase/ptgKinematics.go

```diff
@@ -78,9 +71,11 @@ func wrapWithPTGKinematics(
 		b.Name().ShortName(),
 		logger,
 		baseMillimetersPerSecond,
-		baseTurningRadius,
-		0, // pass 0 to use the default refDist
+		options.AngularVelocityDegsPerSec,
```

Comment:
I don't like how this function is developing.  Can we refactor it so it becomes something like `NewPTGFrameFromKinematicOptions` instead?  This should allow it to scale better

Reply:
Yeah, it's kinda been growing. I'll update to do that.

Reply:
Well, not really actually. If we want `kinematicbase` to be able to call `tpspace`, then `tpspace` cannot import `kinematicbase` to receive the options struct.

Reply:
I think that means these options are inherent to tpspace not the kinematic wrapper and then should live in that package.